src/415D90.c

```diff
@@ -865,7 +880,189 @@ void func_802A4A10(void) {
+                D_802AD628[i] = id = hud_element_create(D_802AD640[i]);
+                hud_element_set_scale(id, 0.45f);
+                hud_element_set_flags(id, HUD_ELEMENT_FLAGS_FILTER_TEX | HUD_ELEMENT_FLAGS_80);
+            }
+
+            D_802AD618 = id = hud_element_create(HES_AnimatedHandPointer);
+            hud_element_set_flags(id, HUD_ELEMENT_FLAGS_DROP_SHADOW | HUD_ELEMENT_FLAGS_80);
+            hud_element_set_render_pos(id, D_802AD63C, D_802AD63E);
+
+            D_802AD61C = id = hud_element_create(HES_GreenArrowUp);
+            hud_element_set_flags(id, HUD_ELEMENT_FLAGS_DROP_SHADOW | HUD_ELEMENT_FLAGS_80);
+            hud_element_set_render_pos(id, D_802AD63C + 39, D_802AD63E - 7);
+
+            D_802AD620 = id = hud_element_create(HES_GreenArrowDown);
+            hud_element_set_flags(id, HUD_ELEMENT_FLAGS_DROP_SHADOW | HUD_ELEMENT_FLAGS_80);
+            hud_element_set_render_pos(id, D_802AD63C + 39, D_802AD63E + 78);
+
+            D_802AD614 = 10;
+            x = D_802AD63C;
+            y = D_802AD63E;
+            set_window_properties(6, x, y, 144, (D_802AD60A * 13) + 26, 0, func_802A5290, NULL, -1);
+            set_window_properties(7, x + 18, y - 6, 108, 16, 1, func_802A56F8, NULL, -1);
+            x = 20;
+            y = 186;
+            set_window_properties(8, x, y, 280, 32, 0x14, func_802A5738, NULL, -1);
+            set_window_update(6, 1);
+            set_window_update(7, 1);
+            set_window_update(8, 1);
+            D_802AD604 = 1;
+            break;
+        case 1:
+            if (D_802AD607 == 0) {
+                D_802AD606 = D_802AD605;
+                if (battleStatus->currentButtonsHeld & BUTTON_STICK_UP) {
+                    if (D_802AD605 > 0) {
+                        D_802AD605--;
+                    } else if (battleStatus->currentButtonsPressed & BUTTON_STICK_UP) {
+                        D_802AD605--;
+                    }
+                }
+
+                if (battleStatus->currentButtonsHeld & BUTTON_STICK_DOWN) {
+                    if (D_802AD605 < D_802AD66C - 1) {
+                        D_802AD605++;
+                    } else if (battleStatus->currentButtonsPressed & BUTTON_STICK_DOWN) {
+                        D_802AD605++;
+                    }
+                }
+
+                if (D_802AD605 < 0) {
+                    D_802AD605 = D_802AD66F - 1;
+                }
+                if (D_802AD66C - 1 < D_802AD605) {
+                    D_802AD605 = 0;
+                }
+
+                if (D_802AD606 != D_802AD605) {
+                    hud_element_set_scale(D_802AD628[D_802AD606], 0.45f);
+                    sfx_play_sound(SOUND_MENU_CHANGE_SELECTION);
+                }
+
+                if (D_802AD605 < D_802AD608) {
+                    D_802AD608 = D_802AD605;
+                }
+                if (D_802AD605 >= D_802AD609) {
+                    D_802AD608 = D_802AD605 - ((u8) D_802AD60A - 1);
```

Comment:
Potentially `D_802AD60A` is typed wrongly?

Reply:
I was unable to fix it by changing the type, unfortunately